hub/src/server/authentication.ts

```diff
@@ -182,22 +186,23 @@ export class V1Authentication {
    * Returns [] if there is no association token, or if the association token has no scopes
    */
   getAuthenticationScopes(): Array<AuthScopeType> {
-    let decodedToken: TokenType
-    try {
-      decodedToken = decodeToken(this.token)
-    } catch (e) {
-      logger.error(this.token)
-      logger.error('getAuthenticationScopes')
-      throw new ValidationError('Failed to decode authentication JWT')
-    }
+    const payload = (() => {
```

Comment:
it may clean things up a bit to add a new function `decodeAndGetPayload` -- there's three uses of it.

Reply:
Implemented in https://github.com/blockstack/gaia/pull/245/commits/435b6b76171ce426e70075b1f6632d97876161ca